package exception;

public class ExceptionTask {
	public static void main(String[] args) {
		//try~catch 사용하기
		//무한반복 안에서 5개의 정수만 입력받고 5칸 배열에 넣기
	
		
	
	}
}
